code/script.js

```diff
@@ -255,28 +249,25 @@ const generateBoard = () => {
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-  console.log(secret)
```

Comment:
you can delete the console.log after it is no more needed.

Reply:
When it's red it already delete :)